website/docs/components/badge-count/partials/code/showcase.md

```diff
@@ -1,7 +1,7 @@
 <section data-test-percy data-section="showcase">
```

Comment:
I mostly left this alone as per https://github.com/hashicorp/design-system/blob/main/wiki/Website-Editing-existing-content.md but made some minor updates to get it to work better and make the headers use correct numerical order.